android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -56,6 +58,19 @@ public void onMapReady(GoogleMap map) {
                     CameraPosition.builder().target(location).zoom(15f).bearing(0).tilt(0).build()));
 
             mMap = map;
+
+            mMap.getUiSettings().setMapToolbarEnabled(true);
+            mMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
+                @Override
+                public boolean onMarkerClick(Marker marker) {
+
+                    Intent mapIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(
+                            "http://maps.google.com/maps?saddr=51.5, 0.125&daddr="+ location.latitude + ", " + location.longitude));
```

Comment:
@arp95 As mentors, we expect all the students to do some research on how to proceed doing some task. Right now I can see that you haven't actually tried finding a way on how to proceed with this. Try leaving saddr blank and see what happens. 

@all : NEVER EVER HARD CODE ANYTHING!!